added cfw-calender

nullx002 opened the pull request:
support for cfw-calendar added

wasamasa commented:
One big thing sticking out: You haven't stuck the theme block into the correct position. C doesn't come after W...

nullx002 commented:
tried few colours for the foreground but none worked like yellow, also removing it altogether is not helping with the highlight. moved code to C, didn't knew it was ordered alphabetically so added it just below previous code that i added. also merge and rebase works slightly differently in my version of magit than you explained but finally i figured it out after a bit of experimentation. also learned a few more things about git in the process. thanks for help.

wasamasa commented on gotham-theme.el:
```diff
@@ -4,6 +4,7 @@
 
 ;; Author: Vasilij Schneidermann <v.schneidermann@gmail.com>
 ;; URL: https://github.com/wasamasa/gotham-theme
+;; Package-Version: 20170414.1046
```
You shouldn't touch this either.

nullx002 commented:
sorry, done.

wasamasa commented:
OK, one more thing: The commit message doesn't reflect your changes, please reword it to something like "Add calfw faces".

nullx002 commented:
ok, done

wasamasa commented:
Great, thanks!
